### ISSUE TITLE
Activate watchtower client by default

### ISSUE THREAD

**Issue by AbelLykens** [lightningnetwork/lnd#5344]
### Background

I think it would be useful if the watchtower client is enabled by default. More people should use watchtowers I feel; but them having to activate wtclient first makes it harder. Why not enable by default?

### Your environment

* version of `lnd` commit=v0.12.1-beta

### Steps to reproduce

People try to connect to watchtower
People get error `watchtower not active`

### Expected behaviour

People try to connect to watchtower
Watchtower connects

### Actual behaviour

Well I just said that ;).

**Comment by DarthBenro008** [comment]
Hey @Roasbeef @carlaKC , I would like to work on this issue 😃 !

**Comment by saubyk** [comment]
> Personally, I dont think we should activate it by default. It would mean that users who dont want to use watchtowers or users who dont even know about them will have backups continue to build up because the user would not know that they need to add the details of a tower to send the updates too. I think it should remain an opt-in feature.

Agreed. Unless there's a possibility that, the backups start building **only** after the user adds the tower to send the updates to, it wouldn't make sense to activate WT client by default.

**Comment by saubyk** [comment]
Closing this issue for now, as not much dev activity on this and no user demand as well.
